### PR TITLE
Clarify 'usable registered mDNS hostname'

### PR DESCRIPTION
**Pull request by jeroendeb**
PR for #94

### CODE REVIEW

File: draft-ietf-rtcweb-mdns-ice-candidates.md [hunk 1]
```diff
@@ -157,8 +157,9 @@ described below.
    whether an address is safe to expose. If so, expose the candidate and abort
    this process.
 
-2. Check whether the ICE agent has a usable registered mDNS hostname resolving
-   to the ICE candidate's IP address. If one exists, skip ahead to Step 6.
+2. Check whether the ICE agent has previously generated, registered and stored
+   a mDNS hostname for this IP address per steps 3, 4 and 6. If one exists,
```
Comment (tQsW) [inline draft-ietf-rtcweb-mdns-ice-candidates.md]: > a mDNS

an mDNS

Comment (jeroendeb) [reply draft-ietf-rtcweb-mdns-ice-candidates.md]: Ok, changed.

File: draft-ietf-rtcweb-mdns-ice-candidates.md [hunk 2]
```diff
@@ -157,8 +157,9 @@ described below.
    whether an address is safe to expose. If so, expose the candidate and abort
    this process.
 
-2. Check whether the ICE agent has a usable registered mDNS hostname resolving
-   to the ICE candidate's IP address. If one exists, skip ahead to Step 6.
+2. Check whether the ICE agent has previously generated, registered and stored
+   an mDNS hostname for this IP address as per Steps 3, 4 and 6. If it has, skip
```
Comment (juberti) [inline draft-ietf-rtcweb-mdns-ice-candidates.md]:
```suggestion
   an mDNS hostname for this IP address as per Steps 3, 4, and 6. If it has, skip
```

File: draft-ietf-rtcweb-mdns-ice-candidates.md [hunk 3]
```diff
@@ -157,8 +157,9 @@ described below.
    whether an address is safe to expose. If so, expose the candidate and abort
    this process.
 
-2. Check whether the ICE agent has a usable registered mDNS hostname resolving
-   to the ICE candidate's IP address. If one exists, skip ahead to Step 6.
+2. Check whether the ICE agent has previously generated, registered and stored
```
Comment (juberti) [inline draft-ietf-rtcweb-mdns-ice-candidates.md]:
```suggestion
2. Check whether the ICE agent has previously generated, registered, and stored
```

**Pull request merged**